src/common/libschedutil/ops.h

```diff
@@ -17,6 +17,13 @@
 extern "C" {
 #endif
 
+/* SET_R_NULL - R is not needed by the caller.  To avoid unnecessary
+ * KVS lookups, do not lookup R.  Instead pass a NULL for R to the
+ * callback */
+enum schedutil_free_flags {
+    SCHEDUTIL_FREE_FLAG_SET_R_NULL = 1,
```

Comment:
Suggest we have one enum for all flags.
Also, I'm not crazy about that flag name.  Would something like this work?
```c
enum schedutil_flags {
    SCHEDUTIL_FREE_NOLOOKUP = 1, // ops->free() will be called with R=NULL
};
```